VimeoNetworking/Sources/Models/VIMVideo.h

```diff
@@ -171,6 +171,15 @@ typedef NS_ENUM(NSUInteger, VIMVideoProcessingStatus) {
 - (void)setIsLiked:(BOOL)isLiked;
 - (void)setIsWatchLater:(BOOL)isWatchLater;
 - (BOOL)hasReviewPage;
+/**
+ Determines if the video has a file transfer page. A video has a file
+ transfer page if @p canDownload privacy is @p true and there is a
+ @p file_transfer response.
+ 
+ 
+ @return @p true if the video has a file transfer page.
+ */
+- (BOOL)hasVideoTransferPage;
```

Comment:
Nit: Since this is referred to as `fileTransfer` on the API, maybe call this `hasFileTransferPage`?